Updated all *.yaml files in the deployment folder for the API version apps/v1

DhruvSondhi opened the pull request:
i have updated all the .yaml files for the api version apps/v1 ... all work as intended and has been checked and verified by me on the cloud console ... happy to help ... cheers :)

DhruvSondhi committed:
Updated auth.yaml for apps/v1 version of API

added selector too

DhruvSondhi committed:
Updated frontend.yaml for apps/v1 API version

added selector too

DhruvSondhi committed:
Updated hello-canary.yaml for apps/v1 API version

added selector too

DhruvSondhi committed:
Updated hello-green.yaml for apps/v1 API version

added selector configuration

DhruvSondhi committed:
Updated hello.yaml for apps/v1 API version

added selector configuration

DhruvSondhi commented:
@googlebot I signed it!

timonvw commented on kubernetes/deployments/hello.yaml:
```diff
@@ -1,9 +1,13 @@
   replicas: 3
+  selector:
+    matchLabels:
+    app: hello
```
Typo:

```
selector:
  matchLabels:
    app: hello
    track: stable
```

DhruvSondhi replied:
hello i have it below this comment .. can u confirm the same ?

DhruvSondhi commented on kubernetes/deployments/hello.yaml:
```diff
@@ -1,9 +1,13 @@
+  selector:
+    matchLabels:
+    app: hello
+    track: stable
```
here

debarghya472 commented:
This repository is been used in [Managing Deployments Using Kubernetes Engine](https://google.qwiklabs.com/focuses/639?parent=catalog) lab. It would be highly convenient if this issue is resolved as quickly as possible because I believe students will be facing problem with creating deployments using this deprecated api.

Glad to help:))

DhruvSondhi commented:
i do think so #21 is a very similar pull request ... please confirm the same

DhruvSondhi commented:
i am very happy that this pull request is getting recognised ... i am very happy to help and contribute for the same :)

SakshamJain7 commented:
Your Update is Very heplful ..Thanks a lot

esoterik-dev committed:
Removed field 'track' to fix error

The field 'track' is not recognized and gives error so it is removed.

$ kubectl create -f deployments/hello.yaml
error: error validating "deployments/hello.yaml": error validating data: [ValidationError(Deployment.spec.selector): unknown field "app" in io.k8s.apimachinery.pkg.apis.meta.v1.LabelSelector, ValidationError(Deployment.spec.selector): unknown field "track" in io.k8s.apimachinery.pkg.apis.meta.v1.LabelSelector]; if you choose to ignore these errors, turn validation off with --validate=false

DhruvSondhi commented:
> Your Update is Very heplful ..Thanks a lot

thanks for those kind words ... hope this reaches all people who are stuck with this issue... cheers :)

DhruvSondhi committed:
Merge pull request #1 from Xnehaxixh/master

Removed field 'track' to fix error

DhruvSondhi commented:
please can someone confirm that the files in this PR work as intended ?

SakshamJain7 commented:
> please can someone confirm that the files in this PR work as intended?

Yes , They all working perfectly

DhruvSondhi commented:
> > please can someone confirm that the files in this PR work as intended?
> 
> Yes , They all working perfectly

thanks alot bro for the update ... i have committed one change ... hope it works for all now ... thanks to @Xnehaxixh too for the update :)

DhruvSondhi commented:
@googlebot I fixed it

DhruvSondhi commented:
@googlebot I fixed it.

DhruvSondhi commented:
@googlebot I fixed it.

DhruvSondhi commented:
@googlebot i fixed it !
